src/native/corehost/hostmisc/pal.unix.cpp

```diff
@@ -945,6 +945,11 @@ bool pal::getenv(const pal::char_t* name, pal::string_t* recv)
     return (recv->length() > 0);
 }
 
+bool pal::fullpath(pal::string_t* path, bool skip_error_logging)
+{
+    return realpath(path, skip_error_logging);
+}
+
 bool pal::realpath(pal::string_t* path, bool skip_error_logging)
```

Comment:
I think this deserves a comment - especially since the implementation is "the other way round" from Windows.